src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -232,28 +232,32 @@ public function findDescendantsByType(array $ancestryDbIds, string $entityType,
     }
 
     /**
-     * @param int[]    $ancestryDbIds
-     * @param string   $entityType
+     * @param int[] $ancestryDbIds
+     * @param string $entityType
+     * @param DateTimeImmutable $now
      * @param int|null $limit
-     * @param int      $offset
+     * @param int $offset
      * @return array
      */
-    public function findStreamableDescendantsByType(array $ancestryDbIds, string $entityType, ?int $limit, int $offset) : array
+    public function findStreamableDescendantsByType(array $ancestryDbIds, string $entityType, DateTimeImmutable $now, ?int $limit, int $offset) : array
```

Comment:
I would call this cutoffTime to be in line with the rest of the stuff

Reply:
But it's not the cutoff time. I could do the 7 day logic in the Service I suppose. Then pass it through?